### ISSUE TITLE
Some punctuation marks are HTML-encoded when posted to Bluesky

### ISSUE THREAD

**Issue by michaelShriver** [mauricerenck/mastodon-to-bluesky#11]
Version: Release 1.2.0

Posts to Mastodon that have certain punctuation characters appear on Bluesky HTML-encoded:

<img width="577" alt="Screenshot 2024-06-17 at 1 36 16 PM" src="https://github.com/mauricerenck/mastodon-to-bluesky/assets/26631647/75f15d50-5d09-4e3f-8082-c893e2647305">

Test post to Mastodon:

`Testing some punctuation , . " ? < > ' !`

Appearance when it is cross-posted to Bsky:

`Testing some punctuation , . &quot; ? &lt; &gt; &#39; !`

**Comment by alien-sunset** [comment]
yep, noticed this on my posts as well, this combined with the lack of line breaks/spaces makes some posts very difficult to read.
